docs/community_resources.md

```diff
@@ -19,12 +19,10 @@ If you are looking to stay up to date with the last Conan news, follow us on Twi
 
 - [Updatable Recipes](https://github.com/qchateau/conan-center-bot): Automatically scans available recipes and checked for new upstream releases and tests one configuration
   - The results can be found here: https://qchateau.github.io/conan-center-bot/#/updatable
-- [Pending Review](https://github.com/prince-chrismc/conan-center-index-pending-review)
-  - The results can be found here: https://prince-chrismc.github.io/conan-center-index-pending-review/
 - [System Package Checks](https://github.com/bincrafters/system-packages-checks): Builds automatically all `system` versions of recipes merged on CCI
 and being pull requested on a selection of Linux distributions and FreeBSD
   - The results can be found here: https://bincrafters.github.io/system-packages-checks/
 
 ## Tools
 
-- [Bincrafters Conventions](https://github.com/bincrafters/bincrafters-conventions): Automatically updates Conan recipes to the latest conventions and rules
+- [Bincrafters Conventions](https://github.com/bincrafters/bincrafters-conventions): Automatically updates Conan recipes to the latest conventions and rules
```

Comment:
I'd remove this too

Reply:
Addressed by f8a8983cd8978b81ffeca2f10855044cdd06913b